.teamcity/src/main/kotlin/common/CommonExtensions.kt

```diff
@@ -159,7 +159,7 @@ fun BuildType.applyDefaultSettings(
         if (this@applyDefaultSettings.type != BuildTypeSettings.Type.COMPOSITE) {
             executionTimeoutMin = timeout
         }
-        testFailure = false
+        testFailure = true
```

Comment:
🤔 how does this change interact with cancellation settings?

Will a test failure in QFL abort all later stages of a RfR build or do they run to completion as they do now?